src/pipeline/rackhd/docker/Build.groovy

```diff
@@ -0,0 +1,41 @@
+package pipeline.rackhd.docker
+
+def build(String manifest_path, String clone_dir, String workdir){
+    def ret_dict=[:]
+    node(build_docker_node){
+            deleteDir()
+            def current_workspace = pwd()
+            String library_dir = "./on-build-config"
+            def share_method = new pipeline.common.ShareMethod()
+            share_method.checkoutOnBuildConfig(library_dir)
+            withCredentials([
+                usernamePassword(credentialsId: 'ff7ab8d2-e678-41ef-a46b-dd0e780030e1',
+                                 passwordVariable: 'SUDO_PASSWORD',
+                                 usernameVariable: 'SUDO_USER')]){
+                    timeout(90){
+                        withEnv(["WORKSPACE=${current_workspace}"]){
+                            sh """#!/bin/bash -ex
+                            bash $library_dir/src/pipeline/rackhd/docker/build.sh \
+                                   --MANIFEST_FILE_URL  ${manifest_path} \
+                                   --ARTIFACTORY_URL ${ARTIFACTORY_URL} \
+                                   --STAGE_REPO_NAME ${STAGE_REPO_NAME} \
+                                   --CLONE_DIR ${clone_dir} \
+                                   --IS_OFFICIAL_RELEASE ${IS_OFFICIAL_RELEASE} \
+                                   --WORKDIR ${workdir}
+                            """
+                        }
+                    }
+                    archiveArtifacts 'rackhd_docker_images.tar, build_record'
+                    stash name: 'docker', includes: 'rackhd_docker_images.tar, build_record'
+                    env.DOCKER_STASH_NAME="docker"
+                    env.DOCKER_STASH_PATH="rackhd_docker_images.tar"
+                    env.DOCKER_RECORD_STASH_PATH="build_record"
+                    ret_dict["DOCKER_STASH_NAME"]="docker"
+                    ret_dict["DOCKER_STASH_PATH"]="rackhd_docker_images.tar"
+                    ret_dict["DOCKER_RECORD_STASH_PATH"]="build_record"
```

Comment:
The content of build_record is the output of ```docker load -i rackhd_docker_images.tar```
I think we can get the tag of docker from the tar file.
So there is no need to stash the file build_record and pass it to downstream steps.